### ISSUE TITLE
path_selector breadcrumbs don't work for keyboards

### ISSUE THREAD

**Issue by johrstrom** [OSC/ondemand#4689]
The path_selector breadcrumbs don't work for keyboards. Clicks work, but keyboards send a request to the batch connect form i.e., `/pun/sys/dashboard/batch_connect/<router>/session_contexts/new`. The response is `406 Unacceptable` because it's requesting json on that API.

Obviously it should be sending requests to the files controller, not the batch connect controller.
